src/common/fiscal-year.test.ts

```diff
@@ -1,27 +1,163 @@
 import { DateTime } from 'luxon';
-import { fiscalYear } from '.';
-import { fullFiscalQuarter } from './fiscal-year';
+import {
+  endOfFiscalYear,
+  expandToFullFiscalYears,
+  fiscalQuarter,
+  fiscalYear,
+  fiscalYears,
+  fullFiscalQuarter,
+  fullFiscalYear,
+  startOfFiscalYear,
+} from './fiscal-year';
+import { DateInterval } from './temporal';
 
-describe('fiscal-year', () => {
-  //TODO - This is an example as to what a single test case WOULD be like with a
-  // standard 'it' statement.  However, due to the nature of this function it should also be
-  // transformed into the case below using the 'it.each' to easily cover all cases
-  it('should return current year when date is in second quarter', () => {
-    const regularDate = DateTime.fromObject({ year: 2020, month: 6, day: 1 });
+const feb2019 = DateTime.fromObject({ year: 2019, month: 2, day: 1 });
+const sep302019 = DateTime.fromObject({ year: 2019, month: 9, day: 30 });
+const oct2019 = DateTime.fromObject({ year: 2019, month: 10, day: 1 });
+const nov2019 = DateTime.fromObject({ year: 2019, month: 11, day: 1 });
+const dec2019 = DateTime.fromObject({ year: 2019, month: 12, day: 1 });
+const dec312019 = DateTime.fromObject({ year: 2019, month: 12, day: 31 });
+const jan2020 = DateTime.fromObject({ year: 2020, month: 1, day: 1 });
+const mar312020 = DateTime.fromObject({ year: 2020, month: 3, day: 31 });
+const apr2020 = DateTime.fromObject({ year: 2020, month: 4, day: 1 });
+const jun2020 = DateTime.fromObject({ year: 2020, month: 6, day: 1 });
+const jun302020 = DateTime.fromObject({ year: 2020, month: 6, day: 30 });
+const jul2020 = DateTime.fromObject({ year: 2020, month: 7, day: 1 });
+const sep2020 = DateTime.fromObject({ year: 2020, month: 9, day: 1 });
+const sep302020 = DateTime.fromObject({ year: 2020, month: 9, day: 30 });
+const oct2020 = DateTime.fromObject({ year: 2020, month: 10, day: 1 });
+const dec312020 = DateTime.fromObject({ year: 2020, month: 12, day: 31 });
+const sep302021 = DateTime.fromObject({ year: 2021, month: 9, day: 30 });
+const fy2020 = DateInterval.fromObject({
+  start: oct2019,
+  end: sep302020,
+});
```

Comment:
Small but 
```suggestion
const sep302021 = DateTime.local(2021, 9, 30);
const fy2020 = DateInterval.fromDateTimes(oct2019, sep302020);
```

Reply:
updated consts to use local and fromDateTimes functions